Documentation/AppDiscovery.md

```diff
@@ -17,8 +17,8 @@ This dialog displays a list of applications which can be profiled by Detox Instr
 
 If your app does not appear in the app selection dialog, the following tips will help you troubleshoot.
 
-- Make sure you have integrated your app with the Profiler framework. See [Profiler Framework Integration Guide](Documentation/XcodeIntegrationGuide.md) for more information.
+- Make sure you have integrated your app with the Profiler framework. See [Profiler Framework Integration Guide](/Documentation/XcodeIntegrationGuide.md) for more information.
 - Devices should be connected on the same network (Wi-Fi, LAN for macOS & Simulators or Lightning cable for direct connection)
 - Security software, such as Antivirus and Firewall applications, may interfere with Bonjour; please consult the software manual or contact your system administrator
 - Corporate networks may block or otherwise prevent Bonjour from working; please contact your system administrator
-  - As a workaround, you can connect your device locally to your Mac using a Lightning cable, which will create a Bonjour-supported direct connection.
\ No newline at end of file
+  - As a workaround, you can connect your device locally to your Mac using a Lightning cable, which will create a Bonjour-supported direct connection.
```

Comment:
What has changed here?

Reply:
Probably, line ending. I've edited this file with Github interface.

Reply:
👍